JuicingJuicer/solvd-automation
# Web Automation HW

# Test Cases:
### 1. Test City Weather
1. Open Yahoo home page - Yahoo home page loaded
2. Read the name of the city in the Weather block - City name is read
3. Click the 'See More' link in the Weather block - Weather page is loaded
4. Read the name of the city in the top left corner of the weather page - City name is read
5. Verify city names - City names are matching

### 2. Test Movie Search
1. Open Yahoo home page - Yahoo home page loaded
2. Search for a movie based on given keyword - Movie page loaded with the correct movie searched
3. Read movie properties (title, release date, director) - Movie properties are read
4. Verify movie properties - Movie properties match with expected results

### 3. Test Horoscope Consistency
1. Open Yahoo home page - Yahoo home page loaded
2. Click entertainment link - Yahoo entertainment page loaded
3. Read currently selected horoscope - Currently selected horoscope is read
4. Click life link - Yahoo life page loaded
5. Read currently selected horoscope - Currently selected horoscope is read
6. Verify currently selected horoscopes - Currently selected horoscopes are matching

### 4. Test Country Switch
1. Open Yahoo home page - Yahoo home page loaded
2. Click 'More...' link - Yahoo Products and Services page loaded
3. Click 'Choose your Country' link - Yahoo International page loaded
4. Click 'France' link - France Yahoo home page loaded
5. Read 'Trending Now' text in France Yahoo home page loaded  - Read the French version of 'Trending Now'
6. Verify current language - French version of 'Trending Now' matches with expected results

### 5. Test Crypto Info
1. Open Yahoo home page - Yahoo home page loaded
2. Click finance link - Yahoo finance page loaded
3. Click crypto link - Yahoo crypto page loaded
4. Search for crypto based on given crypto name - Crypto is searched and found
5. Click on the cryptocurrency that was found - Crypto Quote page loaded
5. Read cryptocurrency properties (name, start date) - Cryptocurrency properties are read
6. Verify cryptocurrency properties - Cryptocurrency properties match with expected results
